src/extension.ts

```diff
@@ -69,13 +70,14 @@ export async function activateInternal(ctx: vscode.ExtensionContext, perfStats:
             targetPopulation = tas.TargetPopulation.Team;
         } else if (/alpha/ig.test(extensionVersion.value)) {
             targetPopulation = tas.TargetPopulation.Insiders;
+        } else if (/Insiders/ig.test(vscode.env.appName)) {
```

Comment:
This is changed in the same way as https://github.com/microsoft/vscode-azuretools/pull/1067, but that is in the new `@microsoft/vscode-azext-utils` package, which won't make it in to here until we do the T2 upgrade in #3407. Rather than publish a new deprecated `vscode-azureextensionui` I just copied the code here.